backend/xivdpscalc/character/jobs.py

```diff
@@ -52,32 +55,41 @@ def dh_buffs(cls):
         return {cls.BV, cls.BARD_DH, cls.DEVILMENT}
 
     @classmethod
-    def raid_buffs(cls):
+    def damage_buffs(cls):
         """ Lists damage buffs """
         return {cls.DIV, cls.TRICK, cls.BROTHERHOOD, cls.BARD_DMG, cls.TECH, cls.DEVOTION, cls.EMBOLDEN}
 
     def avg_buff_effect(self, job):
         """ Calculates the average impact of buffs considering its uptime and magnitude """
-        total = 0
-        if self.name == 'EMBOLDEN':  #pylint: disable=comparison-with-callable
+        if self.name == 'EMBOLDEN':   # pylint: disable=comparison-with-callable
             if job == Jobs.RDM or job.role in {Roles.TANK, Roles.MELEE, Roles.RANGED}:
                 decay_interval = 4
                 decay_rate = 0.2
-                for i in range(self.duration / decay_interval):
-                    total += self.multiplier * (1 - decay_rate * i) * decay_interval / self.cooldown
-                return total
+                decay_ticks = self.duration_sec // decay_interval
+                intervals = (self.multiplier * (1 - decay_rate * tick) * decay_interval / self.cooldown_sec
+                             for tick in range(decay_ticks))
+                return sum(intervals)
             return 0 # Sucks to not have Embolden apply, I guess
-        return self.multiplier * self.duration / self.cooldown
+        return self.multiplier * self.duration_sec / self.cooldown_sec
 
 class Jobs(Enum):
-    """ Job modifiers from https://www.akhmorning.com/allagan-studies/modifiers/ """
+    """
+    Contains job related info.
+
+    job_mod: The bonus given to the main stat.
+    role: The Role of the job
+    raidbuff: A list of all raidbuffs that the job has
+
+    job modifiers from https://www.akhmorning.com/allagan-studies/modifiers/
+    note: tanks use STR for damage
+    """
     SCH = (115, Roles.HEALER, [Buffs.CHAIN])
```

Comment:
We need a type annotation for this. Probably something like:

```python
JobMod = int
JobInfo = Tuple[JobMod, Roles, List[Buffs]]
```

Reply:
this breaks the program so i left it out